docs/src/Table.doc.js

```diff
@@ -1269,4 +1269,79 @@ card(
   />,
 );
 
+card(
+  <Example
+    id="sortableExample"
+    name="Example: Sortable header cells with sticky column"
+    defaultCode={`
+    function SortableHeaderExample() {
+      const [sortOrder, setSortOrder] = React.useState('desc');
+      const [sortCol, setSortCol] = React.useState('name');
+
+      const onSortChange = (col) => {
```

Comment:
The example sorting doesn't work. Could we add a few more columns to trigger the scrollbar automatically?

Reply:
Made the update to add more columns and leave off the actual sorting